pkgs/development/interpreters/php/default.nix

```diff
@@ -228,6 +232,7 @@ let
         pdo_mysqlSupport = config.php.pdo_mysql or true;
         libxml2Support = config.php.libxml2 or true;
         apxs2Support = config.php.apxs2 or (!stdenv.isDarwin);
+        embedSupport = config.php.embed or false;
```

Comment:
Maybe this could be `true` as default so there would be no need for another top-level php attribute?